indico/modules/events/editing/models/editable.py

```diff
@@ -117,6 +117,10 @@ def locator(self):
     def event(self):
         return self.contribution.event
 
+    @property
+    def verbose_title(self):
```

Comment:
Not sure if that's the best name. How about `log_title` since it's not suitable outside logging (no i18n + quoting that would probably look strange elsewhere)